src/Chirp.Core/Helpers/DBFacade.cs

```diff
@@ -10,12 +10,17 @@ namespace Chirp.Core.Helpers
 {
     public class DBFacade : IDisposable
     {
-        private const string _sqlDBFilePath = "../Chirp.Core/Assets/chirp.db";
+        // "../Chirp.Core/Assets/chirp.db"
+        private string _sqlDBFilePath = "../Chirp.Core/Assets/chirp.db";
         private SqliteConnection? _SQLite;
         public DBFacade()
+        {
+            InitializeConnection();
+        }
+
+        private void InitializeConnection()
         {
             var fullPath = Path.GetFullPath(_sqlDBFilePath);
-            Console.WriteLine($"Database full path: {fullPath}");
 
             if (!File.Exists(fullPath))
```

Comment:
Maybe we should catch this exception somewhere?

Reply:
Agreed. I can see I forgot to update that in this pull request